[release] 20220121 (#767)
subquery/subql: packages/cli/CHANGELOG.md

## [Unreleased]

## [0.19.0] - 2022-01-21
### Added

> - Allow use `subql build` self contained script from `exports` fields under`package.json` (#753)

Reviewer: ```suggestion
- Allow use `subql build` self contained js as instructed by `exports` fields under`package.json` (#753)
```
